### DISCUSSION TITLE
Changes to icon

### DISCUSSION THREAD

**Discussion by Zigimun** [fullstackhero/blazor-starter-kit#381] [Q&A]
Does anyone know how to change the bar icon.
![image](https://user-images.githubusercontent.com/72990798/147233483-603875fc-491f-48d1-b63f-79f0cfe48945.png)

To something else... I just cannot where this is implemented am I stupid or what :)

Thanks for help

**Comment by ajakblackgoat** [comment]
No code changes required. Simply add or replace  with your own favicon.ico in Client\wwwroot below:
![Snag_d08e731](https://user-images.githubusercontent.com/4223633/147316361-e3fb95db-bd31-48fe-a768-7bfa7dd0a8fb.png)

**Reply by neozhu** [reply]
and clean cache in browser

**Reply by Zigimun** [reply]
Thank you :)
